doc/user/pimv6.rst

```diff
@@ -395,7 +418,24 @@ General multicast routing state
 
    Display current bsr, its uptime and last received bsm age.
 
-.. clicmd:: show ipv6 pim bsrp-info [vrf NAME] [json]
+.. clicmd:: show ipv6 pim bsr candidate-bsr [vrf NAME] [json]
```

Comment:
Where is this command defined? I cant find it here.

Reply:
added, and refactored the show command for v4/v6 while I was at it.